dags/oup/oup_process_file.py

```diff
@@ -0,0 +1,9 @@
+import airflow
+from airflow.decorators import dag
+
+
+@dag(schedule_interval=None, start_date=airflow.utils.dates.days_ago(0))
+def oup_process_file():
+    pass
```

Comment:
Will be implemented on the other PR. We need to have this DAG because of the trigger_file_processing task in harvesting DAG. trigger_file_processing is the last task in harvesting DAG, the task passes files for parsing
https://github.com/cern-sis/workflows/blob/d3f99b5c294a81a64b7d96c1841dec6ddb067fc5/dags/oup/oup_pull_ftp.py#L42-L51